nextflow.config

```diff
@@ -0,0 +1,193 @@
+/*
+ * -------------------------------------------------
+ *  nf-core/deepvariant Nextflow config file
+ * -------------------------------------------------
+ * Default config options for all environments.
+ * Cluster-specific config options should be saved
+ * in the conf folder and imported under a profile
+ * name here.
+ */
+
+ // Global default params, used in configs
+ params {
+
+   container = 'nfcore/deepvariant:dev'
+
+   help = false
+   outdir = 'results'
+   email = false
+
+   //BAM files
+   bam=""
+   bam_folder=""
+   bam_file_prefix=""
+   getBai=false
+
+   // Reference genomes
+   genomes_base = 's3://deepvariant-data/genomes'
+   //genome = 'hg19' //if no fasta specified default of hg19 will be used (see main.nf)
```

Comment:
Please uncomment and set to `false` if following earlier comment in `main.nf` about not setting a default for the genome.